include/ts/apidefs.h.in

```diff
@@ -324,7 +324,7 @@ typedef enum {
       accept connections. This is *not* guaranteed to be called
       before the first connection is accepted.
 
-      Event: TS_EVENT_LIFECYCLE_PORTS_READY_HOOK
+      Event: TS_EVENT_LIFECYCLE_PORTS_READY
```

Comment:
Did you mean to drop the _HOOK here?

Reply:
yes, this is a typo as the event is here https://github.com/apache/trafficserver/blob/master/include/ts/apidefs.h.in#L452